dev/devbuild.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-gprbuild -j0 -r -p -P `dirname $0`/../alr_env.gpr
+gprbuild -j0 -r -p -P `dirname $0`/../alr_env.gpr "$@"
```

Comment:
I don't understand this change.

Reply:
Mmm yes, this is unrelated, just to be able to pass extra args to this script. I was fighting a "-fPIC" error but in the end the problem was elsewhere.